src/pool/rpc.h

```diff
@@ -366,6 +370,16 @@ CRT_RPC_DECLARE(pool_get_acl, DAOS_ISEQ_POOL_GET_ACL, DAOS_OSEQ_POOL_GET_ACL)
 
 CRT_RPC_DECLARE(pool_prop_set, DAOS_ISEQ_POOL_PROP_SET, DAOS_OSEQ_POOL_PROP_SET)
 
+#define DAOS_ISEQ_POOL_ACL_UPDATE	/* input fields */	 \
```

Comment:
(style)  Macros with complex values should be enclosed in parentheses

Reply:
The usual false positive for adding CaRT RPCs. Ignoring.